src/elm/Page/Community/Objectives.elm

```diff
@@ -151,13 +151,16 @@ viewObjective ({ shared } as loggedIn) model community index objective =
                         )
                     ]
                 ]
-            , div [ class "flex" ]
+            , div [ class "flex justify-between" ]
                 [ a
-                    [ class "w-full button button-secondary button-sm mr-10 hidden md:flex md:visible"
+                    [ class "button button-secondary button-sm w-full sm:w-40 mb-4 mx-2 px-4 hidden"
```

Comment:
This button is always hidden due to the `hidden` class. Maybe we don't need this button at all?